### DISCUSSION TITLE
How to display animated GIFs?

### DISCUSSION THREAD

**Discussion by pazzotranquillo** [gickowtf/pixoo-homeassistant#66] [Q&A]
Hi guys,

I started using this component yesterday and I really love it!

The question I have is about animated GIFs. How can I display them correctly? I've used the  type: image component and unfortunately the picture appears to be static on the Divoom.

Thanks in advance for your support!

Regards

**Comment by SB2DD** [comment]
Animated GIFs aren't currently supported, although it might be possible to add them. There is an [api endpoint](https://docin.divoom-gz.com/web/#/5/54) that allows gifs to be played, but it would be a little bit complicated to properly add it with files/base64 like we have now with the images component. Also, gifs wouldn't work in the "components" pages. But, it is theoretically possible to add support for a gif page_type! I'll test this later today.

**Reply by pazzotranquillo** [reply]
That would be great! Thanks.

**Comment by gickowtf** [comment]
#68 is merged in Version 1.11.0
